src/containers/pages/FocusInvestigation/single/index.tsx

```diff
@@ -466,25 +466,25 @@ const mapStateToProps = (state: Partial<Store>, ownProps: any): DispatchedStateP
       InterventionType.FI,
       [PlanStatus.COMPLETE],
       CASE_TRIGGERED
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
     completeRoutinePlansArray: getPlansArray(
       state,
       InterventionType.FI,
       [PlanStatus.COMPLETE],
       ROUTINE
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
     currentReactivePlansArray: getPlansArray(
       state,
       InterventionType.FI,
       [PlanStatus.ACTIVE, PlanStatus.DRAFT],
       CASE_TRIGGERED
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
     currentRoutinePlansArray: getPlansArray(
       state,
       InterventionType.FI,
       [PlanStatus.ACTIVE, PlanStatus.DRAFT],
       ROUTINE
-    ),
+    ).filter(p => (jurisdiction ? p.jurisdiction_id === jurisdiction.jurisdiction_id : true)),
```

Comment:
I think we can do the filtering in the selector, ama?

Reply:
Good call